### PR TITLE
docs(drash/v2.x): Add response time service docs

### PR DESCRIPTION
**Pull request by ebebbington**

### CODE REVIEW

File: docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Response Time
+
+This service will record how long a a response takes for any given request, by setting a `X-Response-Time` header on the response.
```
Comment (crookse) [inline docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md:3]:
```suggestion
This service will record how long a response takes for any given request by setting a `X-Response-Time` header on the response.
```

**Review by crookse — CHANGES_REQUESTED** [review]

File: docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+- [Configuration](#configuration)
+  - [Content-Security-Policy](#content-security-policy)
+  - [Expect-CT](#expect-ct)
+  - [Referrer-Policy](#referrer-policy)
+  - [Strict-Transport-Security](#strict-transport-security)
+  - [X-Content-Type-Options](#x-content-type-options)
+  - [X-DNS-Prefetch-Control](#x-dns-prefetch-control)
+  - [X-Frame-Options](#x-frame-options)
+  - [X-Powered-By](#x-powered-by)
+  - [X-XSS-Protection](#x-xss-protection)
```
[the remark this answers is not in the document]
Comment (ebebbington) [reply docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md:22]: looooool

[the remark this answers is not in the document]
Comment (ebebbington) [reply docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md]: https://github.com/drashland/website-v2/pull/41/commits/94cf532c9a0694172edf9cd21e95e15ce25c69dc

File: docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md (hunk 1 above)
Comment (ebebbington) [reply docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md]: https://github.com/drashland/website-v2/pull/41/commits/fcaa402e7af8f8391bce96bdf3e4c17c4ca9ea5f

File: docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+class HomeResource extends Drash.Resource {
+
+  public paths = ["/"];
+
+  public GET(request: Drash.Request, response: Drash.Response): void {
+    return response.text("Hello, world!");
+  }
+}
+
+// Create and run your server (with PaladinService instantiated)
```
[the remark this answers is not in the document]
Comment (ebebbington) [reply docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/5_response_time.md]: https://github.com/drashland/website-v2/pull/41/commits/fcaa402e7af8f8391bce96bdf3e4c17c4ca9ea5f

**Comment by ebebbington** [comment]
my bad, didnt even realise you suggested commits until i did the changes

**Comment by crookse** [comment]
lmfao

**Review by crookse — APPROVED** [review]

**Pull request merged**